src/element.ts

```diff
@@ -198,29 +199,29 @@ export class Element {
    * element.drawWithStyle();
    * ```
    */
-  setStyle(style: ElementStyle | undefined): this {
-    this.style = style;
+  setStyle(style: ElementStyle): this {
+    this.style = {...this.style, ...style};
```

Comment:
This is a major and breaking change that I'm not sure has been adequately discussed as far as the implications and definitely not adequately documented. It's probably okay but how would someone remove a key from this.style using the interface now? This seems to me to be a new method like "updateStyle" or something. I would not expect a method called "setStyle" to retain attributes from the previous style.

Reply:
Agree, reverted. Same visual results.